criticality_score/generate.py

```diff
@@ -48,34 +48,40 @@ def main():
     parser.add_argument(
         "--sample-size",
         type=int,
-        default=1000,
+        default=5000,
```

Comment:
Just curious -- is there some rationale for this number (or the old number)? Any star threshold is going to mean that stars can potentially affect the criticality score, which seems a little sad since criticality score is supposed to go beyond stars.

Obviously, they're not independent, though. It seems like the default sample size should be set based on the count, though exactly how might vary by language. A thought: plot something like "likelihood that the top projects ranked by stars are in the top projects by criticality score" (based on the data you have so far), fit that curve, and set the sample size based on where the likelihood approaches zero.

Reply:
Yes it is sad, but we are constantly scaling up, trying to process more repos so that we can do this better. github api is slow and has constant rate limits, so even a few thousand takes more a day.

It does not impact criticality score algo, which was the main purpose for project launch. We want organizations to generate criticality scores for their own oss usage, with their own infra.